Tests/KafkaTests/KafkaConsumerTests.swift

```diff
@@ -82,4 +83,43 @@ final class KafkaConsumerTests: XCTestCase {
             )
         }
     }
+
+    func testConsumerStatistics() async throws {
+        let uniqueGroupID = UUID().uuidString
+        var config = KafkaConsumerConfiguration(
+            consumptionStrategy: .group(id: uniqueGroupID, topics: ["this-topic-does-not-exist"]),
+            bootstrapBrokerAddresses: []
+        )
+
+        var metricsOptions = KafkaConfiguration.KafkaMetrics()
+
+        let handler = MockTimerHandler()
```

Comment:
There is a `MetricsTestingKit` which we should use here instead so we don't need the `MockTimerHandler`

Reply:
Good to know, let me use it